jenkins/Makefile

```diff
@@ -8,21 +8,28 @@
 #  image-ci:           builds the CI image
 #  image-fedora:       builds the Fedora-based development image
 #  image-rpmbuild-all: builds all the default containers for building RPMs
-#  image-rpmbuild-rhel-7:  builds a single RPM build container (e.g., for RHEL-7)
+#  image-rpmbuild-rhel-7:  builds a single RPM build container (e.g. for RHEL-7)
 #  push-ci:            pushes the CI image to the registry
 #  push-fedora:        pushes the Fedora-based development image to the registry
-#  push-rpmbuild-all:  pushes all the default RPM build containers to the registry
+#  push-rpmbuild-all:  pushes all default RPM build containers to the registry
 #
 
 # Include definition of _DISTROS
 include ../utils/utils.mk
 
-# Default list of distributions for which to build containers in which to build RPMs
+# Default list of distributions for which to build containers in which to build
+# RPMs
 RPMBUILD_BASEIMAGE_DEFAULTS = \
 	rhel-9 rhel-8 rhel-7 \
-	fedora-36 fedora-35 fedora-34 \
+	fedora-36 fedora-35 \
 	centos-9 centos-8
 
+# All Fedora images are based on Fedora 35
+FEDORA_BASE_IMAGE_REF = quay.io/fedora/fedora:35
+
+# All UBI images are based on UBI 8.5
+UBI_BASE_IMAGE_REF = registry.access.redhat.com/ubi8/ubi-init:8.5
```

Comment:
I think these comments are misleading...e.g., these configs are not used for building rpmbuild-containers.

Rather than "all" I think it would be better to enumerate the uses:  "The CI and Fedora development container images are based on Fedora 35" and "The UBI development container image is based on UBI 8.5".  (Although, it would probably be better to avoid including the version number in the comment, since it is clearly specified in the code line; instead just describe the symbols' uses:  "Base image for CI and Fedora development container image" and "Base image for UBI development container image".)